doc/design/blob-uuid-1.md

```diff
@@ -0,0 +1,47 @@
+A blob is referenced by a key generation function. the value of this key is used to index the blob in the database table of identities and on the s3 storage. `s3://dandiarchive/blob/<3digitskey>/<next3digitskey>/<restofkey>`
```

Comment:
```suggestion
A blob is referenced by a key generation function. the value of this key is used to index the blob in the database table of identities and on the s3 storage. `s3://dandiarchive/blob/<3digitskey>/<next3digitskey>/<key>`
```
this would be consistent with the form used below in upload 1.1, and the form I would generally prefer even though at the little cost of duplicating first 6 digits in the directory names and the key name.  I find such a form easier to reuse later on and possibly avoid the file name to start with `-` if it happens to be the 7th digit in the key.

Reply:
but I do not feel too strong about it (I do not immediately see where using `restofkey` would really fail) so feel free to reject but then adjust path below in upload 1.1 to match.